src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -198,7 +196,7 @@ internal void EnqueueBindToDataSource(MirrorData mirrorData)
 
         #region Public Class Properties
 
-        internal bool IsDataBound { get { return mirrorData != null; } }
```

Comment:
I added the other note below without seeing yours. Basically this property is weird now (it is in fact set externally), and we should clean it up subsequently when there's more time to test, for now let's keep it and open a new task to track this clean-up work.